windows/win_file.ps1

```diff
@@ -50,6 +50,52 @@ $result = New-Object psobject @{
     changed = $FALSE
 }
 
+# Used to delete symlinks when powershell version is 4 or older
+$symlink_util = @"
+using System;
+using System.Runtime.InteropServices;
+
+namespace Ansible.Command
+{
+    public class DeleteSymLink
+    {
+        [DllImport("kernel32.dll", CharSet=CharSet.Unicode, SetLastError=true)]
+        public static extern bool RemoveDirectory(string lpPathName);
+    }
```

Comment:
Should probably have a managed wrapper around this that does the return-val check / `Marshal.GetLastWin32Error()` and passes it into/throws a `Win32Exception`.